lib/mongo/applicants.ts

```diff
@@ -182,12 +182,24 @@ export const getApplicantsForCommittee = async (
 
         applicant.optionalCommittees = [];
 
-        if (new Date(period.applicationPeriod.end) > new Date()) {
-          applicant.name = "Skjult ";
+        const today = new Date();
+        const sevenDaysAfterInterviewEnd = new Date(period.interviewPeriod.end);
+        sevenDaysAfterInterviewEnd.setDate(
+          sevenDaysAfterInterviewEnd.getDate() + 7
+        );
+
+        if (
+          new Date(period.applicationPeriod.end) > today ||
+          today > sevenDaysAfterInterviewEnd
+        ) {
+          applicant.owId = "Skjult";
+          applicant.name = "Skjult";
+          applicant.date = today;
```

Comment:
`today` her er vel ikke riktig?

Reply:
> `today` her er vel ikke riktig?

Var for å skjule når brukeren søkte, da applicant.date er et Date objekt

Reply:
> Hvorfor ikke bare ha undefined eller no. Og trenger vi å skjule det uansett?

Tja, vi bruker jo egt ikke infoen, men var hvis vi evnt skulle vist når brukere har søkt, og hvilke tider som passer for dem evnt i fremtiden